frontend/src/pages/groups/GroupList.tsx

```diff
@@ -0,0 +1,114 @@
+import { FunctionComponent, useMemo } from 'react'
+import { useSortBy, useTable } from 'react-table'
+import cx from 'classnames'
+import LayoutWithNav from '../../layouts/LayoutWithNav'
+import groupsFixture from '../../fixtures/groups'
```

Comment:
Request (non-blocking): please add an issue for loading this from the server

Reply:
👍  https://github.com/distributeaid/shipment-tracker/issues/46